src/simtools/applications/derive_pulse_shape_parameters.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+r"""Derive Gaussian sigma and exponential tau from rise/fall widths.
+
+This application solves for (sigma, tau) of a Gaussian convolved with a
+causal exponential such that the resulting pulse matches the requested
+rise and fall width definitions. It uses the solver implemented in
+``simtools.simtel.pulse_shapes.solve_sigma_tau_from_risefall``.
+
+Inputs:
```

Comment:
Inputs / outputs is exactly what you describe in the 'Command line arguments' section? Maybe not needed